modules/ROOT/pages/single-sign-on-with-saml.adoc

```diff
@@ -93,7 +93,7 @@ edit the authenticationManager-config.properties as follows:
         logout.link.hidden=true
 ----
 +
-Make sure to link:multi-tenancy-and-tenant-configuration#toc2[set the right tenant admin username].
+Make sure to link:multi-tenancy-and-tenant-configuration[set the right tenant admin username].
```

Comment:
This paragraph does not exist anymore, as the multi-tenancy is now deprecated. I am not sure of what to do with this link